receiver/mongodbreceiver/integration_test.go

```diff
@@ -79,6 +79,7 @@ var (
 
 func TestMongodbIntegration(t *testing.T) {
 	t.Run("Running mongodb 4.0", func(t *testing.T) {
+		t.Skip("Test fails and wasn't caught due to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17037")
```

Comment:
Can you please file a separate issue to follow up. I'd like to close https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17037 :)

Reply:
And I think it's flaky. It passed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17040

Reply:
Added https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17070

Reply:
If the test is flakey I would change the description here, and add the link to #17070

Reply:
> If the test is flakey I would change the description here, and add the link to #17070

Done!